src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,47 @@
-export const PeopleFilters = () => {
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
```

Comment:
you can make it short
```suggestion
import cn from 'classnames';
```